docs/core/events.md

```diff
@@ -96,20 +38,65 @@ The main `eventCategory` you can subscribe to are:
 These events are triggered from the `state` package after a block is committed. You can get the
 full list of `event` categories [here](https://godoc.org/github.com/tendermint/tendermint/types#pkg-constants).
 
-The `type` and `attribute` value of the `query` allow you to filter the specific `event` you are looking for. For example, a `transfer` transaction triggers an `event` of type `Transfer` and has `Recipient` and `Sender` as `attributes` (as defined in the [`events` file of the `bank` module](https://github.com/cosmos/cosmos-sdk/blob/master/x/bank/types/events.go)). Subscribing to this `event` would be done like so:
+The `type` and `attribute` value of the `query` allow you to filter the specific `event` you are looking for. For example, a `MsgEthereumTx` transaction triggers an `event` of type `ethermint` and has `sender` and `recipient` as `attributes`. Subscribing to this `event` would be done like so:
 
 ```json
 {
     "jsonrpc": "2.0",
     "method": "subscribe",
     "id": "0",
     "params": {
-        "query": "tm.event='Tx' AND transfer.sender='senderAddress'"
+        "query": "tm.event='Tx' AND ethereum.recipient='hexAddress'"
     }
 }
 ```
 
-where `senderAddress` is an address following the [`AccAddress`](../basics/accounts.md#addresses) format. 
+where `hexAddress` is an Ethereum hex address (eg: `0x1122334455667788990011223344556677889900`).
+
+### Ethereum JSON-RPC Events
+
+<!-- TODO: PublicFiltersAPI -->
```

Comment:
I will leave this for another PR along with the REST server

Reply:
👍